package.json

```diff
@@ -157,6 +157,9 @@
       "pre-commit": "lint-staged"
     }
   },
+  "resolutions": {
```

Comment:
What does this do?

Reply:
That seems to be for Yarn's [selective dependency resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) feature. But we use npm, which does not support that. Unless we'd use a package like https://www.npmjs.com/package/npm-force-resolutions.

The question is: what kind of issues do we have with the regular inferred version of the scheduler package?

Reply:
I was just debugging an issue and it seemed like resolving a conflict between 0.18 and 0.19 coming from two different packages was a potential solution but it turned out to be a different problem. I removed this for now but let's keep an eye on it.